### ISSUE TITLE
efficientnet stuck at first epoch

### ISSUE THREAD

**Issue by karim1871** [ZFTurbo/segmentation_models_3D#36]
Hi, thank you for the great work, I have an issue with efficient net.

segmentation-models-3D version: 1.0.6
TensorFlow version: 2.8.0
Keras version: 2.8.0

Efficientnet v1 and v2 model trainings don't start, they are stuck at epoch 1. 

Image size (96,96,96,1) as float32,  mask :  type, Y_patches_categorical obtained with Y_patches_categorical = to_categorical(Y_patches, num_classes=2). 

Other models work fine.
